### PR TITLE
Converted movies into cards

### PR DESCRIPTION
**Pull request by dustxd**
Placed movie posters into cards 👀

Screen Shot 2021-06-14 at 11 18 55 PM

### CODE REVIEW

**Review by ronan-f — APPROVED** [review]
10/10 pr

**Comment by mostlyfabulous** [comment]
7/7 would ship it

File: components/Movie.js [hunk 1]
```diff
@@ -1,23 +1,48 @@
...
+        </Card>
     )
 }
 
-const handleError = (e) => {
-    const defaultImgLocation = "/assets/default-poster.jpeg"
-
-    e.target.src = defaultImgLocation
-}
+export default withStyles(styles)(Movie);
```
Comment (mostlyfabulous) [inline components/Movie.js:48]: This is what I was talking about when I mentioned there's 3 APIs to apply styles in MUI. Is this your preferred way?
https://material-ui.com/styles/basics/

**Review by mostlyfabulous — APPROVED** [review]

File: components/Movie.js (hunk 1 above)
Comment (dustxd) [reply components/Movie.js:48]: LOL yeah I know, there's also just inline styling `style={{ color: 'red' }}` 👀 I do tend to use this way to style but hooks can't do this so idk 🤷

**Pull request merged**